recipe/meta.yaml

```diff
@@ -13,22 +15,32 @@ source:
 
 build:
   number: 1005
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - python
+    - pip
     - setuptools
+    - wheel
   run:
     - python
     - psutil >=3.2.0
```

Comment:
This is optional since version 2.9.2, and it's not in the `setup.py`, you could move it to `run_constrained`

```suggestion
```

See: https://github.com/search?q=repo%3Arasbt%2Fpyprind%20psutil&type=code

Reply:
`psutils` imported here https://github.com/rasbt/pyprind/blob/57d8611ae86cc2cb71d6f1ab973476fc9bea5b7a/pyprind/prog_class.py#L20

Reply:
It's technically an optional dependency because of the `try except`. If the build doesn't fail after removal, I'd rather not include it as a `run` dependency. I would also consider adding it to the `run_constrained` section.

Reply:
Of interest, `psutil` is fine in `run_constrained` but is required for the `monitor` tests.

I'll remove the pinning as there doesn't seem to be any mention of `3.2.0` (or anything else) elsewhere.

Reply:
Then it can be moved it to `test/required`

Reply:
They made `psutils ` optional since version `2.9.2`, it's always imported between `try-catch`.

See: https://github.com/rasbt/pyprind/blob/881fa94ef2da4acebcd33686fc258f42ca7066fa/CHANGELOG.md?plain=1#L59-L63

Reply:
added the pinning back in as it is declared in `requirements.txt`